SECURITY.md

```diff
@@ -0,0 +1,43 @@
+# Security Policies and Procedures
+
+This document outlines security procedures and general policies for the `Open Build Service`
+project.
+
+  * [Reporting a Bug](#reporting-a-bug)
+  * [Disclosure Policy](#disclosure-policy)
+  * [Comments on this Policy](#comments-on-this-policy)
+
+## Reporting a Bug
+
+The `Open Build Service` team and community take all security bugs in `Open Build Service` seriously.
+Thank you for improving the security of `Open Build Service`. We appreciate your efforts and
+responsible disclosure and will make every effort to acknowledge your
+contributions.
+
+Report security bugs by emailing the [Security Team](https://www.suse.com/support/security/contact/) at security@suse.com.
```

Comment:
How about: 

The OBS as openSUSE project makes use of the [SUSE/openSUSE security team](https://www.suse.com/support/security/contact/). So please report security problems by contacting them.

Reply:
I agree, steps to follow are set by SUSE. Their instructions or contact email could change and we wouldn't have to change our document.

Reply:
after working 5 years in the security team, IMO, it is easier the contact website to change/go offline than the e-mail address ```security@suse.com```

Reply:
Sorry, I deleted my previous comment accidentally. It was:

> I agree, steps to follow are set by SUSE. Their instructions or contact email could change and we wouldn't have to change our document.

But @vpereira, yours is a good reason too :thinking:

Reply:
@hennevogel  are we good to go here?

Reply:
If you tell me the encrypting the mail does not matter so much, sure :-)